scripts/echo_gitignore.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env sh
+
+echo '#------------------------------------------------------------------------------#'
```

Comment:
Just how I separate my code.
I helps me finding groups of code and commenting/documenting it with less lines very effectively.
For example, I use

```latex
%-------------------------------------------%
% setup documentclass

\documentclass[
    ...
]{...}

\do_sth{...}

%-------------------------------------------%
% begin document

\begin{document}
...
```

Here, it looks a bit verbose since there are only like 3 lines, but in a real document, it helps me.